frontend/test/unit/specs/components/v-popover.spec.js

```diff
@@ -147,20 +143,26 @@ describe("VPopover", () => {
           props: { popoverProps: { trapFocus: false, autoFocusOnHide: false } },
         })
         await doOpen()
+        expectOpen()
         await clickOutside()
         expect(getTrigger()).not.toHaveFocus()
       })
     })
   })
 
   describe("hideOnClickOutside", () => {
-    it("should hide the popover if a click happens outside the popover by default", async () => {
+    // This test is broken (for some reason clickOutside does not appear to actually cause a click
+    // to happen in this case).
+    // https://github.com/WordPress/openverse/issues/2220
+    // eslint-disable-next-line jest/no-disabled-tests
+    it.skip("should hide the popover if a click happens outside the popover by default", async () => {
```

Comment:
```suggestion
    it("should hide the popover if a click happens outside the popover by default", async () => {
```
The fix has been merged.